cluster/Vagrantfile

```diff
@@ -32,6 +32,7 @@ k8s_ver = ENV['CONTIV_K8S_VERSION'] || DEFAULT_K8S_VERSION
 orc_path = case k8s_ver
            when /^v1\.[45]\./ then 'k8s1.4/'
            when /^v1\.[67]\./ then 'k8s1.6/'
+           when /^v1\.[89]\./ then 'k8s1.8/'
```

Comment:
I would probably just have this match 1.8 for now... otherwise people might try 1.9 (and we'll allow them to!) before we've verified that 1.9 actually works 😉